L/libserialport/build_tarballs.jl

```diff
@@ -1,18 +1,19 @@
 using BinaryBuilder
 
-# Collection of sources required to build libserialport
 name = "libserialport"
-version = v"0.1.1"
+version = v"0.2.0"
```

Comment:
> This PR creates a new version 0.2.0 (0.1.1 is the latest release), which builds from master.

As we discussed on Slack, this is not ideal, but the situation of upstream  not releasing a new version for several years is not ideal either.  I just wonder whether it's safer to use v0.1.2 here, since don't exactly know what will be the version number of the next stable version, whenever it'll come.

Reply:
You are right, its quite likely for the next version to be v0.1.2.